qskinny.pro

```diff
@@ -8,7 +8,8 @@ SUBDIRS = \
     tools \
     support \
     examples \
```

Comment:
I could not find any example that make use of state combinations; the only example that uses states at all is the IOT dashboard. However that one sets different hints per state, so nothing we could make use of with the functions here.